apps/wallet/src/ui/app/hooks/useGetAllCoins.ts

```diff
@@ -35,7 +35,8 @@ export function useGetAllCoins(coinType: string, address?: SuiAddress | null) {
 
             return allData;
         },
-        enabled: !!address,
+        enabled: !!address && !!coinType,
+        staleTime: 5000,
```

Comment:
Fine with this approach for now, but we probably should rework this flow in general to do JIT coin fetching and just use the balance instead of using the cached fetch.